templates/zerver/gitlab-error.md

```diff
@@ -0,0 +1,12 @@
+You are using the **GitLab auth backend**, but it is not properly
+configured. Please check the following:
+
+You have added `{{ root_domain_uri }}/complete/gitlab/` as the callback URL
```

Comment:
I think this should start with ``*`` like the other points below?

Reply:
made the ```You have added .........``` part start with `*`. followed the google-error and gitub error pages' format